build_library_with_toolchain.sh

```diff
@@ -40,7 +46,7 @@ configure_options="--prefix=$CMAKE_PREFIX_PATH --disable-shared --enable-static
 # Overwrite/extend for specific packages
 if [ $1 == 'poco' ]; then
     configure_options="--config=Android_static --no-samples --no-tests"
-    export ANDROID_ABI=$abi
+    export ANDROID_ABI
```

Comment:
This variable is already being exported in `config.sh`; perhaps we can just delete this line.